src/providers/export/tensorFlowRecords/tensorFlowHelpers.ts

```diff
@@ -132,6 +132,17 @@ export function textEncode(str: string): Uint8Array {
     return result;
 }
 
+/**
+ * @arr - Input Uint8Array byte array
+ * @description - Get a UTF8 string value
+ */
+export function textDecode(arr: Uint8Array): string {
+    Guard.null(arr);
```

Comment:
what if it's empty?

Reply:
Actually this is not an issue as in any language map on an empty list just do not start the enumeration.